pymatgen/command_line/bader_caller.py

```diff
@@ -231,6 +225,62 @@ def summary(self):
 
         return summary
 
+    @classmethod
+    def from_path(cls, path, suffix=""):
+        """
+        Convenient constructor that takes in the path name of VASP run
+        to perform Bader analysis.
+
+        Args:
+            path (str): Name of directory where VASP output files are
+                stored.
+            suffix (str): specific suffix to look for (e.g. '.relax1'
+                for 'CHGCAR.relax1.gz').
+
+        """
+
+        def _get_filepath(filename):
+            name_pattern = filename + suffix + '*' if filename != 'POTCAR' \
+                else filename + '*'
+            paths = glob.glob(os.path.join(path, name_pattern))
+            fpath = None
+            if len(paths) >= 1:
+                # using reverse=True because, if multiple files are present,
+                # they likely have suffixes 'static', 'relax', 'relax2', etc.
+                # and this would give 'static' over 'relax2' over 'relax'
+                # however, better to use 'suffix' kwarg to avoid this!
+                paths.sort(reverse=True)
+                warning_msg = "Multiple files detected, using %s" \
+                              % os.path.basename(paths[0]) if len(paths) > 1\
+                    else None
+                fpath = paths[0]
+            else:
+                if filename == "CHGCAR":
```

Comment:
Instead of this nested if/else logic, probably easier just to return `None` from this function if the file isn't found, and below write:

```
chgcar_filename = _get_filepath("CHGCAR")
if chgcar_filename is None:
    raise IOError("Could not find CHGCAR!")
```

etc.